with-python-asyncio/app.py

```diff
@@ -0,0 +1,32 @@
+import asyncio
+import time
+import resend
+
+async def send_email(n):
+    params: resend.Emails.SendParams = {
+        "from": "onboarding@resend.dev",
+        "to": ["delivered@resend.dev"],
+        "subject": "hi",
+        "html": "<strong>hello, world!</strong>",
+    }
+
+    email: resend.Email = resend.Emails.send(params)
```

Comment:
It's been a while that I don't use Python asyncio, so I'm not sure if this is correct, but given that the `send` function is not `async`, don't we need to use `asyncio.to_thread` to run the blocking function async?

Reply:
@felipevolpone I updated it to use to_thread since its simpler overall. I was getting the same result with wrapping the blocking fn around a `async def` to turn the blocking function into a coroutine + using `as_completed`. 
We can verify that they're running in parallel because the total runtime is equal to the longest send email request, instead of summing up their request times.